src/components/Auth.jsx

```diff
@@ -6,7 +6,7 @@ import Login from './Login'
 import WebhookQuery from './WebhookQuery'
 
 const Auth = ({ serverSettings, match }) => {
-  if (serverSettings.discord && !serverSettings.user) {
+  if (serverSettings.enabledAuthMethods.length > 0 && !serverSettings.user) {
     if (match.params.category || match.params.lat) {
       localStorage.setItem('params', JSON.stringify(match.params))
     }
```

Comment:
Can't add a comment on line 13 of this file but you need to pass serverSettings as props through <Login /> here as well, like you did in `App.jsx`

Reply:
Done